src/qml/editorwidgets/CheckBox.qml

```diff
@@ -3,6 +3,7 @@ import QtQuick.Controls 2.12
 
 Item {
   signal valueChanged( var value, bool isNull )
+  signal requestGeometry(var item, var layer)
```

Comment:
Could we let these tools inherit from `EditorWidgetBase` instad of `Item` and put this boilerplate code in there?